ui/src/components/accordion2.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
+
+import * as AccordionPrimitive from '@radix-ui/react-accordion';
```

Comment:
Consider using the native `<details>` element with additional group closing handlers instead of adding a dependency on top that is probably already obsolete.

We benefit from using what is already there in HTML.

Reply:
@DarkMenacer thank you for considering this for a future PR.